src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -2425,17 +2501,7 @@ public void Inheritance_04()
             var comp = CreateCompilation(source);
             comp.VerifyDiagnostics();
             var actualMembers = GetProperties(comp, "B").ToTestDisplayStrings();
-            var expectedMembers = new[]
-            {
-                "System.Object B.P1 { get; init; }",
-                "System.Object B.P2 { get; init; }",
-                "System.Object B.P3 { get; init; }",
-                "System.Object B.P4 { get; init; }",
-                "System.Object B.P5 { get; init; }",
-                "System.Object B.P6 { get; init; }",
-                "System.Object B.P7 { get; init; }",
-            };
-            AssertEx.Equal(expectedMembers, actualMembers);
+            AssertEx.Equal(new string[0], actualMembers);
```

Comment:
>string [](start = 31, length = 6)

`P5` should be synthesized per  [https://github.com/dotnet/csharplang/pull/3527](https://github.com/dotnet/csharplang/pull/3527)

Reply:
`P5` is `virtual` in the base class rather than `abstract`.